Reconstruction/RecCalorimeter/tests/options/runFullCaloSystem_SimAndDigitisation.py

```diff
@@ -0,0 +1,296 @@
+saveecalendcaptool.caloHits.Path = "ECalEndcapHits"
+saveecalfwdtool = SimG4SaveCalHits("saveECalFwdHits", readoutNames = [ecalFwdReadoutName])
+saveecalfwdtool.positionedCaloHits.Path = "ECalFwdPositionedHits"
+saveecalfwdtool.caloHits.Path = "ECalFwdHits"
+savehcaltool = SimG4SaveCalHits("saveHCalHits",readoutNames = [hcalReadoutName])
+savehcaltool.positionedCaloHits.Path = "HCalPositionedHits"
+savehcaltool.caloHits.Path = "HCalHits"
+saveexthcaltool = SimG4SaveCalHits("saveExtHCalHits",readoutNames = [extHcalReadoutName])
+saveexthcaltool.positionedCaloHits.Path = "ExtHCalPositionedHits"
+saveexthcaltool.caloHits.Path = "ExtHCalHits"
+savehcalendcaptool = SimG4SaveCalHits("saveHCalEndcapHits", readoutNames = [hcalEndcapReadoutName])
+savehcalendcaptool.positionedCaloHits.Path = "HCalEndcapPositionedHits"
+savehcalendcaptool.caloHits.Path = "HCalEndcapHits"
+savehcalfwdtool = SimG4SaveCalHits("saveHCalFwdHits", readoutNames = [hcalFwdReadoutName])
+savehcalfwdtool.positionedCaloHits.Path = "HCalFwdPositionedHits"
+savehcalfwdtool.caloHits.Path = "HCalFwdHits"
+
+# next, create the G4 algorithm, giving the list of names of tools ("XX/YY")
+from Configurables import SimG4SingleParticleGeneratorTool
+pgun = SimG4SingleParticleGeneratorTool("SimG4SingleParticleGeneratorTool",saveEdm=True,
+                particleName=particleType,energyMin=energy,energyMax=energy,etaMin=-5.0,etaMax=5.0,
+                OutputLevel = DEBUG)
+
+geantsim = SimG4Alg("SimG4Alg",
+                       outputs= ["SimG4SaveCalHits/saveECalBarrelHits", "SimG4SaveCalHits/saveECalEndcapHits", 
+                                 "SimG4SaveCalHits/saveECalFwdHits", "SimG4SaveCalHits/saveHCalHits", 
+                                 "SimG4SaveCalHits/saveExtHCalHits", "SimG4SaveCalHits/saveHCalEndcapHits", 
+                                 "SimG4SaveCalHits/saveHCalFwdHits"],
+                       eventProvider=pgun,
+                       OutputLevel=INFO)
+
+#Configure tools for calo reconstruction
+# EM scale calibration
+from Configurables import CalibrateCaloHitsTool
+calibHcells = CalibrateCaloHitsTool("CalibrateHCal", invSamplingFraction="34.5")
+
+from Configurables import CalibrateInLayersTool
+calibEcalBarrel = CalibrateInLayersTool("CalibrateECalBarrel",
+                                   # sampling fraction obtained using SamplingFractionInLayers from DetStudies package
+                                   samplingFraction = [0.12125] + [0.14283] + [0.16354] + [0.17662] + [0.18867] + [0.19890] + [0.20637] + [0.20802],
+                                   readoutName = ecalBarrelReadoutName,
+                                   layerFieldName = "layer")
+calibEcalEndcap = CalibrateInLayersTool("CalibrateEcalEndcap",
+                                    # sampling fraction obtained using SamplingFractionInLayers from DetStudies package
+                                    samplingFraction = [0.15] * 119, 
```

Comment:
can you correct to the new calib constants?

Reply:
It is done using CalibrateCaloHitsTool, as the sampling fraction is the same for all layers. Just note that the 1/sampling fraction is the parameter in this tool.

Reply:
Sorry, I added this comment before you pushed those changes. Indeed it is ok.